addon/templates/components/bs4/bs-form/element/layout/horizontal.hbs

```diff
@@ -0,0 +1,12 @@
+{{#if hasLabel}}
+  {{component labelComponent labelClass=horizontalLabelGridClass}}
+  <div class="{{horizontalInputGridClass}}">
+    {{yield}}
+    {{component errorsComponent}}
+  </div>
+{{else}}
+  <div class="{{horizontalInputGridClass}} {{horizontalInputOffsetGridClass}}">
+    {{yield}}
+    {{component errorsComponent}}
+  </div>
+{{/if}}
```

Comment:
If I am not missing anything, the only change here (and also in `vertical.hbs`) is the missing feedback icon component. Instead of duplicating the template one could also set `feetbackIconComponent` to `null` in bs4/bs-form/element.js. I think the `{{component}}` helper will then simply render nothing.